be/src/exec/vectorized/hdfs_scanner.cpp

```diff
@@ -229,11 +229,11 @@ void HdfsParquetScanner::update_counter() {
     HdfsScanner::update_counter();
 
 #ifndef BE_TEST
-    COUNTER_UPDATE(_scanner_params.parent->_raw_rows_counter, _stats.raw_rows_read);
+    COUNTER_UPDATE(_scanner_params.parent->_rows_read_counter, _stats.raw_rows_read);
     COUNTER_UPDATE(_scanner_params.parent->_expr_filter_timer, _stats.expr_filter_ns);
     COUNTER_UPDATE(_scanner_params.parent->_io_timer, _stats.io_ns);
     COUNTER_UPDATE(_scanner_params.parent->_io_counter, _stats.io_count);
-    COUNTER_UPDATE(_scanner_params.parent->_bytes_read_from_disk_counter, _stats.bytes_read_from_disk);
+    COUNTER_UPDATE(_scanner_params.parent->_bytes_read_counter, _stats.bytes_read_from_disk);
```

Comment:
nit: how about rename the filed bytes_read_from_disk to bytes_read in hdfs_scanner.h?

Reply:
Good idea.